### PR TITLE
[16.0][MIG] hr_holidays_public: Migration to 16.0

### PR DESCRIPTION
**Pull request by yankinmax**
ref: RD-615

### CODE REVIEW

File: hr_holidays_public/tests/test_holidays_public.py [hunk 1]
```diff
@@ -5,52 +5,50 @@
 from datetime import date
 
 from odoo.exceptions import UserError, ValidationError
+from odoo.tests import tagged
 from odoo.tests.common import TransactionCase
 
 
+@tagged("-at_install", "post_install")
```
Comment (pedrobaeza) [inline hr_holidays_public/tests/test_holidays_public.py]: Please remove this. They are not needed.

**Comment by pedrobaeza** [comment]
Can you please also squash administrative commits as stated in https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests ?

**Review by pedrobaeza — APPROVED** [review]

**Review by cyrilmanuel — APPROVED** [review]

**Review by gurneyalex — APPROVED** [review]

**Comment by rafaelbn** [comment]
This functionality is actually in Odoo Community. Are you sure we want to maintain this module? @OCA/human-resources-maintainers 

https://github.com/odoo/odoo/blob/16.0/addons/hr_holidays/models/hr_leave.py#L1219

imagen

**Pull request merged**